recipes/mdnsd/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+import os
+import sys
+from conans import ConanFile, CMake, tools
+from conans.tools import download, unzip
+from conans.model.version import Version
+
+
+class mdnsdConan(ConanFile):
+    name = "mdnsd"
+    license = "BSD 3-Clause"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/**"
+    ]
+    homepage = "https://github.com/Pro/mdnsd"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Improved version of Jeremie Miller's MDNS-SD implementation"
+    topics = ("dns", "daemon", "multicast", "embedded", "c")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "cpp_compatible": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+        "cpp_compatible": False
+    }
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        if Version(self.version) <= "0.8.0":
+            folder_name = "%s-%s" % (self.name, "0.8")
```

Comment:
That's weird?!

Reply:
Archive is named 0.8 not 0.8.0, thus then a full version is given it fails to extract it. 

I could try giving the package a short version, however I had problems with that when working on open62541.

Reply:
If you give it version `0.8`, then this exception is not needed.
Would this break something?

Reply:
Fixed it in a later commit